### DISCUSSION TITLE
Dbt Artifact version manifeste.json

### DISCUSSION THREAD

**Discussion by Ines70** [OpenLineage/OpenLineage#1870] [Q&A]
I've been working on OpenLineage for a few weeks now and I had a question regarding the handling of metadata, specifically the manifest.json file in dbt. Dbt now uses version 9 of the manifest.json, while in the list of supported versions in the OpenLineage package, the list stops at 7. Is this because OpenLineage currently cannot handle a version higher than 7 for the manifest.json ?

Thank you in advance for your response ☺☺

**Comment by mobuchowski** [comment]
dbt manifest versions are generally not removing anything (with some exceptions) just adding new stuff. We are handling those, just not explicitely.

**Comment by merobi-hub** [comment]
Hello @Ines70 , thank you for your question -- and thanks for contributing to the project! I wanted to welcome you and invite you to join our Slack community if you haven't already: http://bit.ly/OpenLineageSlack. That's a good source for up-to-date information about what's happening in the project -- as well as a good place to ask questions (or better, honestly).
